app/frontend/src/features/profile/view/leaveReference/formSteps/ReferenceStepHeader.tsx

```diff
@@ -1,30 +1,60 @@
 import { Typography } from "@material-ui/core";
+import HeaderButton from "components/HeaderButton";
+import { BackIcon } from "components/Icons";
 import {
+  PREVIOUS_STEP,
   REFERENCE_FORM_HEADING_FRIEND,
   REFERENCE_FORM_HEADING_HOSTED,
   REFERENCE_FORM_HEADING_SURFED,
+  REFERENCE_SUBMIT_HEADING,
 } from "features/profile/constants";
 import { ReferenceType } from "pb/references_pb";
+import { useHistory } from "react-router";
 import { referenceTypeRoute } from "routes";
+import makeStyles from "utils/makeStyles";
 
 export interface ReferenceStepHeaderProps {
-  name: string;
-  referenceType: string;
+  name?: string;
+  referenceType?: string;
+  isSubmitStep: boolean;
 }
 
+const useStyles = makeStyles((theme) => ({
+  header: {
+    alignItems: "center",
+    display: "flex",
+  },
+  title: {
+    marginInlineStart: theme.spacing(2),
+  },
+}));
+
 export default function ReferenceStepHeader({
   name,
   referenceType,
+  isSubmitStep,
```

Comment:
```suggestion
  isSubmitStep = false,
```
Minor stylistic thing/opinionated nitpick: but in React it's pretty idiomatic that the absence of a boolean prop is implied to be false, rather than having to explicitly force the user to do so which clutters the JSX imo